pkg/node/peer_manager/peer_manager.go

```diff
@@ -424,6 +455,22 @@ func (a *PeerManagerImpl) suspended(p peer.Peer) bool {
 	return a.peerStorage.IsSuspendedIP(ip, time.Now())
 }
 
+func (a *PeerManagerImpl) blackListed(p peer.Peer) bool {
+	a.mu.RLock()
+	defer a.mu.RUnlock()
+	ip := storage.IpFromIpPort(p.RemoteAddr().ToIpPort())
+	return a.peerStorage.IsBlackListedIP(ip, time.Now())
+}
+
+func (a *PeerManagerImpl) restrict(p peer.Peer, duration time.Time, reason string) {
```

Comment:
Pass `time.Now()` as method argument instead of unused `duration` argument.

Reply:
done